src/test/resources/features/06_pensions.feature

```diff
@@ -1415,24 +1414,20 @@ Feature: Pensions Journeys - Income Tax Submission
 
     # step to validate change Pension scheme
     When the user clicks on the Change scheme link
-    Then the user should be redirected to the "Did a UK pension scheme pay tax on the short service refunds?" page
-    And the user clicks the continue button
     Then the user should be redirected to the "Pension scheme paying tax on the short service refund" page
     And the user clicks the continue button
     Then the user should be redirected to the "Short service refund summary" page
 
     # step to validate add another pension scheme link
     When the user clicks on the Add another Pensions Scheme Tax Reference link
-    Then the user should be redirected to the "Did a UK pension scheme pay tax on the short service refunds?" page
 
-    When the user selects the yes radio button
-    And the user clicks the continue button
     Then the user should be redirected to the "Pension scheme paying tax on the short service refund" page
     And the user selects the Name of pension scheme field and enters a value of TEST
-    And the user selects the Pension Scheme Tax Reference field and enters a value of 12345678RQ
+    And the user selects the Pension Scheme Tax Reference field and enters a value of 123456
```

Comment:
`the Pension Scheme Tax Reference field` should not be referring to a PSTR anymore as it is QOPS for foreign submissions